xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/localePicker.js

```diff
@@ -30,8 +30,8 @@
 #set ($currentLocale = $services.localization.currentLocale)
 #foreach ($locale in $collectiontool.sort($services.localization.availableLocales, 'displayName'))
   #if ("$!locale" != '')
-    #set ($localeName = $escapetool.xml($stringtool.capitalize($locale.getDisplayName($locale))) +
-      ' <small class="text-muted">(' + $stringtool.capitalize($locale.getDisplayName($currentLocale)) + ')</small>')
+    #set ($localeName = $escapetool.xml($stringtool.capitalize($locale.getDisplayName($locale)) +
+      ' (' + $stringtool.capitalize($locale.getDisplayName($currentLocale)) + ')'))
```

Comment:
So I guess it means a change in the way locales are displayed? Could you share the screenshot before/after?

Reply:
Yes and no. The change happened already in 14.0RC1, I assume due to the upgrade of bootstrap-select in [XWIKI-19323](https://jira.xwiki.org/browse/XWIKI-19323). Therefore, this particular change does not affect the way locales are displayed as the `<small>`-element hasn't been copied by bootstrap-select since the changes in 14.0 RC1. Here the screenshots of the affected admin section in XWiki 13.10.2 and XWiki 14.0 RC1 (i.e., without this change):

![2022-01-28-Localization-XWiki-13 10 2](https://user-images.githubusercontent.com/198317/151526424-149f7106-1786-41ed-beb6-323fe6345850.png)

![2022-01-28-Localization-XWiki-14 0-rc-1](https://user-images.githubusercontent.com/198317/151526451-ab65b8b8-856d-49ef-b388-d8d726ce095a.png)

So if you consider this a regression, the regression had already been introduced before.

Reply:
Ok, thanks for the heads up. So LGTM.